.env.example

```
@@ -0,0 +1,122 @@
+OPENAI_API_KEY=your-openai-api-key-here
+MODEL_NAME=gpt-4o
+
+# ===== Vector Search & Embeddings (TEI) =====
+# Text Embeddings Inference service configuration
+TEI_URL=http://localhost:8080
+TEI_API_KEY=
+ENABLE_VECTOR_STORAGE=true
+VECTOR_STORAGE_ASYNC=true
+MIN_SIMILARITY_THRESHOLD=0.6
+# Model for embeddings - use sentence-transformers models for TEI
+MODEL_EMBEDDING_NAME=sentence-transformers/all-MiniLM-L6-v2
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The MODEL_EMBEDDING_NAME key should go before the TEI_API_KEY key

(UnorderedKey)